Classes/Projection/ProjectorInterface.php

```diff
@@ -34,4 +34,11 @@ public function getReadModelClassName(): string;
      * @api
      */
     public function reset();
+
+    /**
+     * Returns true if the projection maintained by the concreted projector does not contain any data (yet).
+     *
+     * @return boolean
+     */
+    public function isEmpty();
```

Comment:
Again, I struggle a little bit with the name. Is the projection really "empty"? OTOH I can't really think of a better one atm